src/AppInstallerCLICore/Workflows/SourceFlow.cpp

```diff
@@ -107,8 +107,23 @@ namespace AppInstaller::CLI::Workflow
             std::string_view name = context.Args.GetArg(Args::Type::SourceName);
             std::string_view arg = context.Args.GetArg(Args::Type::SourceArg);
             std::string_view type = context.Args.GetArg(Args::Type::SourceType);
+            bool isExplicit = context.Args.Contains(Args::Type::SourceExplicit);
 
-            Repository::Source sourceToAdd{ name, arg, type };
+            Repository::SourceTrustLevel trustLevel = Repository::SourceTrustLevel::None;
+            if (context.Args.Contains(Execution::Args::Type::SourceTrustLevel))
+            {
+                std::vector<std::string> trustLevelArgs = Utility::Split(std::string{ context.Args.GetArg(Execution::Args::Type::SourceTrustLevel) }, '|');
+                std::vector<std::string> trustLevels;

```

Comment:
nit: a vector copy is probably not needed.

```
                for (auto& trustLevelArg : trustLevelArgs)
                {
                    Utility::Trim(trustLevelArg);
                }
```

Reply:
Updated Utility::Split so that it accepts an optional bool parameter for trimming.